packages/frontend/src/home/home.html

```diff
@@ -42,7 +42,12 @@ <h1 class="heading">
             v-bind:title="'Open ' + moduleName + '\'s npm page'"
             :href="'https://www.npmjs.com/package/'+moduleName"
             target="_blank"
-          >{{ moduleName }}</a>
+          >
+          {{ moduleName }}
+          <template v-if="packagesBundleSizesResponse && packagesBundleSizesResponse[moduleName]">
+            <small>({{ packagesBundleSizesResponse[moduleName]["gzip"] }} gzipped)</small>
+          </template>
```

Comment:
(not important) I think the `<template>` can be a `<small>` here (I think you can add `v-if`s anywhere)

Reply:
Ooh, good call, easy adjustment.